satpy/demo/__init__.py

```diff
@@ -142,3 +143,37 @@ def get_hurricane_florence_abi(base_dir='.', method=None, force=False,
     num_frames = int((actual_slice[1] - actual_slice[0]) / actual_slice[2])
     assert len(filenames) == len(channels) * num_frames, "Not all files could be downloaded"
     return filenames
+
+
+def download_typhoon_surigae_ahi(base_dir=SATPY_DEMO_DATA_DIR,
+                                 channels=(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16),
+                                 segments=(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)):
```

Comment:
Do we want to provide a `method=` parameter here that either downloads the files or creates FSFile objects to be passed to the Scene? Maybe a separate PR, but might not be that difficult with what you have here already?

Reply:
I don't think it's needed. The name of the function is `download` which clearly states that a download is performed. Usage of FSFile (which isn't supported by the hsd reader at the moment) doesn't belong in this module, but would rather be showcased in pytroll-examples.